spec/hash_key_and_range_key_lockable_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'pry'
```

Comment:
I think this is left over from some debugging, but just a nit.  It appears in the next file as well